.github/CONTRIBUTING.md

```diff
@@ -5,6 +5,36 @@ appreciated and welcomed. If you're planning a new feature or changing
 the API, please create an issue first. This way we can ensure that your precious
 work is not in vain.
 
+Table of Contents
+
+
+
+- [Issues](#issues)
+- [Your first Contribution](#your-first-contribution)
+- [Setup](#setup)
+	- [Using npm](#using-npm)
+	- [Using yarn](#using-yarn)
```

Comment:
The indentation is not correct. Could you please check with prettier? Or an online linter? https://dlaa.me/markdownlint/

Reply:
Done, pls join the discussion on #850